.travis.yml

```diff
@@ -1,43 +1,69 @@
+addons:
+  artifacts:
+    s3_region: "us-west-2"
+
+dist: trusty
 sudo: false
-language: python
-services:
-  - memcached
-  - mysql
-python:
-  - "2.7"
-  # - "3.6"
-env:
-  matrix:
-    - DB=sqlite3
```

Comment:
we no longer use sqlite3

Reply:
I've removed the sqlite3/mysql config option since we're no longer using mysql - sqlite is now the default.